### ISSUE TITLE
input not change

### ISSUE THREAD

**Issue by lhj0621** [nhn/tui.grid#772]
**Describe the bug**
Double-clicking in the field results in an error.

**Expected behavior**
```var Grid = tui.Grid;

var grid = new Grid({
    el: document.getElementById('insu_grid'), // 컨테이너 엘리먼트
    data: {
        api: {
            readData: {
                url: '/insu/person/' + $('#ssid').val(),
                method: 'POST'
            }
        }
    },
    rowHeaders: [
        {
            type: 'rowNum',
        },
        {
            type: 'checkbox',
        },
    ],
    scrollX: false,
    bodyHeight: 400,
    columns: [{
        header: '영문명',
        name: 'ename',
        editor: 'text',
        valign: 'middle',
        editOptions: {
            type: 'text'
        },
        validation: {
            required: true,
            regExp: /[^ㄱ-ㅎ|ㅏ-ㅣ|가-힣]/g
        },
        sortingType: 'desc',
        sortable: true
    }, {
        header: '생년월일',
        name: 'birth',
        align: 'center',
        valign: 'middle',
        editor: 'text',
        editOptions: {
            type: 'text'
        },
        validation: {
            validation: {
                required: true,
                regExp: /^(19[0-9][0-9]|20\d{2})(0[0-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])$/
            },
        },
        sortingType: 'desc',
        sortable: true
    }, {
        header: '성별',
        name: 'gender',
        align: 'center',
        valign: 'middle',
        editor: {
            type: 'select',
            options: {
                listItems: [
                    { text: '남성', value: 'M' },
                    { text: '여성', value: 'F' },
                ]
            }
        },
        editOptions: {
            type: 'text'
        },
        sortingType: 'desc',
        sortable: true
    }, {
        header: '상태',
        name: 'stts',
        align: 'center',
        valign: 'middle',
        validation: {
            regExp: /^'정상'$/
        },
        width: 250,
        sortingType: 'desc',
        sortable: true
    }],
});

```

**Screenshots**
![](https://github.com/lhj0621/imagetemp/blob/master/2019-10-25%2017;59;12.gif?raw=true)

**Desktop (please complete the following information):**
 window 10
 ie 10
 toast 4.6.1

**Comment by js87zz** [comment]
@lhj0621 
I tested without communicating with API server in `IE10`, but it is operated well.
if you add `detail scenario`, `source` and the `data`, it will be more helpful to us.

**Comment by lhj0621** [comment]
@js87zz 
ok... 
my source and data show you 
```
<!DOCTYPE html>
<html>
<header>
    <script src="https://code.jquery.com/jquery-1.12.3.min.js"></script>
    <script src="https://code.jquery.com/ui/1.11.3/jquery-ui.js"></script>
    <link rel="stylesheet" href="https://uicdn.toast.com/tui-grid/latest/tui-grid.css" />
</header>
<div id='insu_grid'></div>

<script src="https://uicdn.toast.com/tui-grid/latest/tui-grid.js"></script>
<script type="text/javascript">

    const gridData = [
        {
            name: 'test1',
            ename: 'text1',
            jumin: '901025 - 1234562',
            stts: '정상'
        },
        {
            name: 'test2',
            ename: 'text2',
            jumin: '970513-1111111',
            stts: '정상'
        }
    ];

    var Grid = tui.Grid;
    var grid = new Grid({
        el: document.getElementById('insu_grid'), // 컨테이너 엘리먼트  
        data: gridData,
        rowHeaders: [
            {
                type: 'rowNum',
            },
            {
                type: 'checkbox',

            },
        ],
        scrollX: false,
        bodyHeight: 400,
        columns: [{
            header: '이름',
            name: 'name',
            editor: 'text',
            editOptions: {
                type: 'text'
            },
            validation: {
                required: true
            },
            sortingType: 'desc',
            sortable: true
        },
        {
            header: '영문명',
            name: 'ename',
            editor: 'text',
            editOptions: {
                type: 'text'
            },
            validation: {
                required: true,
            },
            sortingType: 'desc',
            sortable: true
        },
        {
            header: '주민등록번호',
            name: 'jumin',
            editor: 'text',
            editOptions: {
                type: 'text'
            },
            validation: {
                required: true
            },
            sortingType: 'desc',
            sortable: true
        },
        {
            header: '상태',
            editor: 'text',
            editOptions: {
                type: 'text'
            },
            validation: {
                required: true
            },
            name: 'stts',
            sortingType: 'desc',
            sortable: true
        },
        ],
    });
</script>

</html>
```

chrome 
![](https://github.com/lhj0621/imagetemp/blob/master/chrometest.gif?raw=true)

IE
![](https://github.com/lhj0621/imagetemp/blob/master/ietest.gif?raw=true)

ie error source
![](https://github.com/lhj0621/imagetemp/blob/master/ieerror.PNG?raw=true)

**Comment by js87zz** [comment]
@lhj0621 
This bug has been caused by using `Number.MAX_SAFE_INTEGER` internally to find an index of the row. Because `IE` has not supported `Number.MAX_SAFE_INTEGER`, not operate well in case. Fortunately, It was fixed in this [PR]() and it will be included next release. Thank you :)

**Comment by js87zz** [comment]
@lhj0621 
fixed in `v4.7.0`
Thank you!
